gdpr.html

```diff
@@ -216,7 +216,15 @@ <h1 style="color: #7bb69d" ><strong>General data protection act (GDPR) is real</
        </div>
 	</div>
 	</div>
- <div align="center"> <img src="img/scroll-anim-black.gif" width="25px"> </div>
+
+	 <svg class="arrows-gdpr">

```

Comment:
I would suggest to use the existing "arrows" class and add a new one for positioning.
For example <svg class="arrows gdpr-arrows-pos">
And then in style.css we would have the class .gpdr-arrows-pos { margin: [margin-top-value] [margin-right-value] [margin-bottom-value] [margin-left-value];}

Reply:
I have used arrows for positioning, and arrows-gdpr path for because paths are taking in parameters of colours and animations as well. Submitting pull request.